packages/ia-topnav/src/primary-nav.js

```diff
@@ -85,20 +85,29 @@ class PrimaryNav extends TrackedElement {
     return this.openMenu === 'search';
   }
 
+  get searchMenu() {
+    if (this.config.hideSearch) {
+      return html``;
+    }
+    return html`
+      <button class="search-trigger" @click="${this.toggleSearchMenu}" data-event-click-tracking="${this.config.eventCategory}|NavSearchOpen">
```

Comment:
Suggestion: line length
`<button 
  class="search-trigger" 
  @click="${this.toggleSearchMenu}"
  data-event-click-tracking="${this.config.eventCategory}|NavSearchOpen"
>
${icons.search}
</button>`